libs/pages/elewa/home/src/lib/components/impact/impact.component.scss

```diff
@@ -0,0 +1,104 @@
+#text-and-image-banner {
+    width: 100vw;
+    min-height: 100vh;
+    display: grid;
+    place-items: center;
+    font-size: var(--elewa-group-website-font-size-elewa-text);
+    font-family: var(--elewa-group-website-dmsans-regular);
+    border: solid 2rem var( --elewa-group-website-team-and-members-color-white);
+    border-bottom-left-radius: 5rem;
+    border-bottom-right-radius: 5rem;
+
+    .title{
+      width: 70%;
+    }
+}
+#text-and-image-banner .banner-container {
+    display: flex;
+    gap: 10%;
+    width: 80%;
+    max-height: 80vh;
+    justify-content: space-between;
+    align-items: center;
+}
+p{
+  padding: 20px 0;
+}
+.button{
+  display: flex;
+  flex-direction: row;
+  padding-top: 60px;
+  .first{
+    margin-right: 30px;
+  }
+
+}
+
+
+$breakpoint-tablet: 768px;
+$breakpoint-low-res-desktops: 1360px;
+
+@media (max-width: $breakpoint-low-res-desktops){
```

Comment:
Would be perfect if breakpoints are saved in a separate scss file but this works for now. Good 👍

Reply:
sure. Thanks